bgpd/bgp_debug.c

```diff
@@ -2611,8 +2611,8 @@ static bool bgp_debug_per_peer(char *host, const struct prefix *p,
 				if (!plist)
 					continue;
 
-				return prefix_list_apply(plist, p) ==
-				       PREFIX_PERMIT;
+				return (prefix_list_apply(plist, p) ==
```

Comment:
Was this applied (especially additional `()` by clang formatter)?

Reply:
Hi Mr. @ton31337 , thanks for your review.
I havent used the clang formatter. 
I tried to combine into a single statement and applied the "()", so that it will be easy to read.
But the frrbott automatically formatted again into multiple lines.

Reply:
Sure, I will remove.

Reply:
As per your suggestion, I corrected the code. thanks Mr. @ton31337 .